src/internal/event-bridge.ts

```diff
@@ -0,0 +1,139 @@
+import { JSONObject } from 'k6'
+import http, { RefinedResponse, ResponseType } from 'k6/http'
+
+import { AWSClient } from './client'
+import { AWSConfig } from './config'
+import { AWSError } from './error'
+import { InvalidSignatureError, SignatureV4 } from './signature'
+import { AMZ_TARGET_HEADER } from './constants'
+import { HTTPHeaders, HTTPMethod } from './http'
+
+type PutEventEntry = {
```

Comment:
Same comment regarding using interface type.

Here's my suggestion with documentation:
```suggestion
/**
 * Represents an event to be submitted.
 *
 * @typedef {Object} PutEventEntry
 *
 * @property {string} Detail - A valid serialized JSON object. There is no other schema imposed. The JSON object may contain fields and nested sub-objects.
 * @property {string} DetailType - Free-form string, with a maximum of 128 characters, used to decide what fields to expect in the event detail.
 * @property {string} EventBusName - The name or ARN of the event bus to receive the event. Only the rules that are associated with this event bus are used to match the event. If you omit this, the default event bus is used.
 * @property {string[]} Resources - AWS resources, identified by Amazon Resource Name (ARN), which the event primarily concerns. Any number, including zero, may be present.
 * @property {string} Source - The source of the event.
 */
interface PutEventEntry {
    Detail: string
    DetailType: string
    EventBusName: string
    Resources: [string]
    Source: string
}
```

What do you think?